pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computeforwardingrule/privateserviceconnectforwardingrule/create.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: compute.cnrm.cloud.google.com/v1beta1
+kind: ComputeForwardingRule
+metadata:
+  labels:
+    label-one: "value-one"
+  name: computeforwardingrule-${uniqueId}
+spec:
+  description: "A VPC private service connect forwarding rule"
+  target:
+    serviceAttachmentRef:
+      name: computeserviceattachment-${uniqueId}
+  location: us-central1
+  networkRef:
+    name: consumernet
+  loadBalancingScheme: ""
```

Comment:
Why do you set a field to empty value? This will be "translated" to field deletion during the update IIRC.

Reply:
Verified that empty value is dropped during resource creation as well. So this field doesn't seem to be used at all.

Reply:
Per TF design([ref](https://github.com/hashicorp/terraform-provider-google/issues/12326)), this field need to be an empty string when target type is service attachment, otherwise it will be default to "EXTERNAL". I can try to collect the http logs to check the existence of this field.

Reply:
Checked the http log and confirmed that with empty string, this field will be dropped. But if we don't have this field, it's will use the default value "EXTERNAL" and we'll get errors from google api: `Error creating ForwardingRule: googleapi: Error 400: Invalid value for field 'resource.loadBalancingScheme': 'EXTERNAL'. Invalid field set in Private Service Connect Forwarding Rule. This field should not be set., invalid`

So I think it's intended to set this field to empty.

Reply:
`EXTERNAL` seems to be a TF-level default value, and we can modify TF logic based on our own needs.

I'd lean towards updating the TF lib to give our customers a better UX, i.e. not having to set `loadBalancingScheme` to an empty string in this use case.

Reply:
Discussed offline, keep using empty string and add a comment asking users to set the field to "" in the PSC resource sample yaml

Reply:
The description was generated form TF lib. There's an [open TF issue ](https://github.com/hashicorp/terraform-provider-google/issues/12326) for TF doc updates. Shall we modify the TF lib to add the field description?